src/js/pages/cart.ts

```diff
@@ -10,6 +10,28 @@ import handleCartAction from '../components/UseHandleCartAction';
 export default () => {
   const {Theme} = window;
   const voucherCodes = document.querySelectorAll(Theme.selectors.cart.discountCode);
+  const cartContainer = document.querySelector<HTMLElement>(Theme.selectors.cart.container);
+
+  if (cartContainer) {
+    cartContainer.addEventListener('click', (event: Event) => {
+      const eventTarget = event.target as HTMLElement;
+
+      if (eventTarget.classList.contains('js-decrement-button')) {
+        const targetItem = eventTarget.closest('.cart__item');
+        const targetValue = targetItem?.querySelector('.js-cart-line-product-quantity') as HTMLElement | null;
+
+        if (targetValue && targetValue.getAttribute('value') === '1' && targetValue.getAttribute('min') === '1') {
```

Comment:
Be careful the min quantity can also be 0 which should be handled like 1, meaning the feature is disabled there is no minimum
Except that I think the original behaviour from classic is consistent and should be used as a base as it's a common way to handle this in many e-commerce shops

tldr; let's stay iso functional

Reply:
Ah and one other comment you seem to be listening to the click event, so only the arrows clicking but the value can also be edited via keyboard, so you should either listen to the `keydown` event as well or maybe easier simply the `change` event?

Unless you need to listen to the specific event to cancel it when the value is not the minimum one?